apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClient.cs

```diff
@@ -1140,8 +1145,11 @@ internal SampleRowKeysStreamImpl(AsyncServerStreamingCall<SampleRowKeysResponse>
             MutateRowsRequest request,
             CallSettings callSettings = null)
         {
+            var originalCallSettings = callSettings;
```

Comment:
Needing both the original call settings and the modified ones is a little odd. I still need to review the rest of the PR to understand what's going on here, but I'd love to find a simpler approach if possible.